src/main/java/com/spring/onlinebookstore/dto/category/CreateCategoryRequestDto.java

```diff
@@ -0,0 +1,17 @@
+package com.spring.onlinebookstore.dto.category;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+
+public record CreateCategoryRequestDto(
+        @NotBlank
+        @Size(min = 2, message = "is too short")
+        @Size(max = 255, message = "is too long")
```

Comment:
If you validate the size for the category it will be great to do the same for book fields ;)